### ISSUE TITLE
[no-restricted-paths] Doesn't support 'path' from tsconfig in zone configuration

### ISSUE THREAD

**Issue by GonzoLiGGa** [benmosher/eslint-plugin-import#1872]
I have configured zone like this:
```
"import/no-restricted-paths": [
      "error",
      {
        "zones": [
          // block imports from libraries source code only index.ts exchange
          {
            "target": "./apps/myFolder/",
            "from": "**@alias**/libs/ui-kit/"
          },
        ]
      }
    ]
```
Where **@alias** is path from **tsconfig.json**.
I have tried to solve problem with typescript import resolver and import alies but it's look like zones configuration doesn't support aliases

**Comment by ljharb** [comment]
It seems reasonable that `from` should pass through this plugin's normal resolver mechanics.

**Comment by nthypes** [comment]
updates on this?

**Comment by ljharb** [comment]
@nthypes nope, it has the "help wanted" label and nobody's submitted a PR.

**Comment by Blackfaded** [comment]
I am facing the same issue, any solutions?

**Comment by Blackfaded** [comment]
Solved it with
```js
settings: {
    'import/resolver': {
      typescript: {
        project: '.',
      },
    },
  },
```

In the rules use the *unaliased* path!
```js
'import/no-restricted-paths': [
      'error',
      {
        zones: [{ target: './src/libs/shared', from: './src/libs/cockpit' }],
      },
    ],
```

**Comment by JounQin** [comment]
close in favor of https://github.com/import-js/eslint-import-resolver-typescript

**Comment by edento** [comment]
Didn't work in a monorepo :( did anyone find a workaround?
